packages/cfd/src/Constants/routes-config.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import CFD from '../Containers';
 import { routes } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-
+import compareCFDs from '../Containers/compare-cfds';
```

Comment:
You can use alias here 
```suggestion
import compareCFDs from 'Containers/compare-cfds';
```